remme/tp/pub_key.py

```diff
@@ -41,8 +46,12 @@
 FAMILY_VERSIONS = ['0.1']
 
 PUB_KEY_ORGANIZATION = 'REMME'
-PUB_KEY_MAX_VALIDITY = timedelta(365)
-PUB_KEY_STORE_PRICE = 10
+
+# Backward compatibility to use more obvious variables name
```

Comment:
No. We don't operate on certificates here, blockchain knows nothing about them.

Reply:
@eugene-babichenko, so `CERTIFICATE_PUBLIC_KEY_MAXIMUM_VALIDITY ` to `PUBLIC_KEY_MAXIMUM_VALIDITY`?